doc/zephyr.doxyfile.in

```diff
@@ -294,7 +294,9 @@ ALIASES                = "kconfig{1}=\c \1" \
                          "pre_kernel_ok=\qualifier pre-kernel-ok" \
                          "async=\qualifier async" \
                          "atomic_api=As for all atomic APIs, includes a full/sequentially-consistent memory barrier (where applicable)." \
-                         "supervisor=\qualifier supervisor"
+                         "supervisor=\qualifier supervisor" \
+                         "driver_api{1}=\brief \htmlonly <span class=\"mlabel\">Driver API</span> \endhtmlonly \1 driver API \
+                                       \details This is the mandatory API any \1 driver needs to expose."
```

Comment:
I think calling this "Driver API" is a bit confusing. These are "driver ops", consumed by the actual driver API. For now, we've kept this internal (INTERNAL_HIDDEN), but I think they should be part of first its own header and then its own Doxygen page. Many consumers of the driver API do not care about how it is implemented behind the scenes, so it just clutters the main page. Also, we need to be careful about using `@copydoc` because there are many situations where there's no 1:1 mapping between the driver API and the op it calls. A simple example is with optional ops: driver API will document `@retval -ENOSYS` if not implemented, whereas the op won't have it.